docs/design/keyboard-shortcuts.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Custom keyboard shortcuts in Office Add-ins
+description: 'Learn how to add custom keyboard shortcuts, also known as key combinations, to your Office Add-in.'
+ms.date: 11/06/2020
+localization_priority: Normal
+---
+
+# Custom keyboard shortcuts in Office Add-ins (preview)
+
+Keyboard shortcuts, also known as key combinations, enable your add-in's users to work more efficiently and they improve the add-in's accessibility for users with disabilities by providing an alternative to the mouse.
+
+> [!IMPORTANT]
+> Keyboard shortcuts are in preview. Please experiment with them in a development or testing environment but don't add them to a production add-in.
+
+Keyboard shortcuts are only supported on Excel and only on these platforms and builds:
+
+* Excel on Windows: Version 2009 (Build 13231.20262)
+* Excel on Mac: 16.41.20091302
+* Excel on the web
+
+> [!NOTE]
+> Keyboard shortcuts work only on platforms that support the following requirement sets. For more about requirement sets and how to work with them, see [Specify Office applications and API requirements](../develop/specify-office-hosts-and-api-requirements.md).
+>
+> - [SharedRuntime 1.1](../reference/requirement-sets/shared-runtime-requirement-sets.md)
+
+There are four major steps to adding keyboard shortcuts to an add-in:
+
+1. Configure the add-in's manifest.
+1. Create or edit the extended overrides JSON file to map action names to keyboard combinations.
```

Comment:
we shouldn't refer to actions as 'action names', let's just use "actions" instead. 

Change "..map action names to keyboard combinations.." to "..map actions to keyboard combinations.."